books.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
Why do you need any placeholder data? The data should be saved in the backend only